packages/orchestrator/lib/clients/processor.integration.test.ts

```diff
@@ -103,10 +103,10 @@ async function processN(handler: (task: TaskAction | TaskWebhook | TaskPostConne
         handler,
         opts: { orchestratorClient, groupKey, maxConcurrency: n, checkForTerminatedInterval: 100 }
     });
+    processor.start();
```

Comment:
The race condition actually occurs if the tasks are created after the processor is started